backends/raspberry_pi/pico2/simple_addmodule_pte.c

```diff
@@ -0,0 +1,38 @@
+__attribute__((aligned(8))) const unsigned char model_pte[] = {
```

Comment:
OK for now, but for the tutorial we should generate --> pte to pte.h --> include.